.vscode/settings.json

```diff
@@ -15,9 +15,12 @@
     ],
     // Set the features to use for cargo commands
     "rust-analyzer.cargo.features": [
-        "frequency-no-relay"
+        "frequency-no-relay",
+        "runtime-benchmarks",
```

Comment:
Adding these targets to cover more code, e.g. benchmarking.rs files do not know about any definitions if `runtime-benchmarks` is not defined.